index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+if (process.env.NEW_RELIC_ENABLED === 'true') {
+  if (!process.env.NEW_RELIC_LICENSE_KEY) {
+    throw new Error('Inconsistent configuration: New Relic enabled but no licence key defined');
```

Comment:
somewhat violent but OK :)
However I would warn about this constraint in the README, and since it is dangerous, use UPPERCASE ;)